effekt/jvm/src/main/scala/effekt/Runner.scala

```diff
@@ -157,6 +157,43 @@ object JSRunner extends Runner[String] {
         createScript(exePath, "node", jsMainFilePath)
     }
 }
+object JSWebRunner extends Runner[String] {
+  import scala.sys.process.Process
+
+  val extension = "js"
+
+  def standardLibraryPath(root: File): File = root / "libraries" / "common"
+
+  override def prelude: List[String] = List("effekt", "option", "list", "result", "exception", "array", "string", "ref")
+
+  def checkSetup(): Either[String, Unit] =
+    Left("Running js-web code directly is not supported (yet). Use `--compile` to generate a js file / `--build` to generate a html file.") // TODO
```

Comment:
```suggestion
    Left("Running js-web code directly is not supported. Use `--compile` to generate a js file / `--build` to generate a html file.")
```
I don't think we want to support this, after all.